Packages/manifest.json

```diff
@@ -1,21 +1,22 @@
 {
   "dependencies": {
-    "com.mlapi.contrib.transport.litenetlib": "https://github.com/Unity-Technologies/mlapi-community-contributions.git?path=/Transports/com.mlapi.contrib.transport.litenetlib#3f0350ae57befd5ea7deabb1db310eb77704f58e",
-    "com.mlapi.contrib.transport.photon-realtime": "https://github.com/Unity-Technologies/mlapi-community-contributions.git?path=/Transports/com.mlapi.contrib.transport.photon-realtime#e47ee18972b110a689c23d6ffc0e5994d18e9eb2",
+    "com.mlapi.contrib.transport.litenetlib": "https://github.com/Unity-Technologies/mlapi-community-contributions.git?path=/Transports/com.mlapi.contrib.transport.litenetlib#develop",
+    "com.mlapi.contrib.transport.photon-realtime": "https://github.com/Unity-Technologies/mlapi-community-contributions.git?path=/Transports/com.mlapi.contrib.transport.photon-realtime#develop",
     "com.veriorpies.parrelsync": "https://github.com/VeriorPies/ParrelSync.git?path=/ParrelSync#bb3d5067e49e403d8b8ba15c036d313b4dd2c696",
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.ai.navigation.components": "https://github.com/Unity-Technologies/NavMeshComponents.git#package",
-    "com.unity.cinemachine": "2.6.3",
-    "com.unity.collab-proxy": "1.3.9",
-    "com.unity.ide.rider": "2.0.7",
-    "com.unity.ide.visualstudio": "2.0.7",
+    "com.unity.cinemachine": "2.7.4",
+    "com.unity.collab-proxy": "1.7.1",
+    "com.unity.ide.rider": "3.0.7",
+    "com.unity.ide.visualstudio": "2.0.11",
     "com.unity.ide.vscode": "1.2.3",
     "com.unity.learn.iet-framework": "1.2.1",
-    "com.unity.multiplayer.mlapi": "https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi.git?path=/com.unity.multiplayer.mlapi#develop",
-    "com.unity.postprocessing": "2.3.0",
-    "com.unity.test-framework": "1.1.24",
-    "com.unity.textmeshpro": "3.0.1",
-    "com.unity.timeline": "1.4.7",
+    "com.unity.multiplayer.tools": "git+ssh://git@github.com/Unity-Technologies/com.unity.multiplayer.tools.git#develop",
```

Comment:
I don't like that this is here, as this repo is still private + is using git ssh instead of https

Reply:
@VALERE91Unity I understand you used this for profiling. Do we still need this or can we add it later once it's available publicly?

Reply:
Technically yes let me try to be 100% sure and if yes I'm gonna push without it